spec/fixtures/files/svg_image.txt

```diff
@@ -0,0 +1 @@
+"<svg width=\"75\" height=\"44\" viewBox=\"0 0 162 88\" fill=\"none\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M9.9 55.245v18.088c0 3.89 1.565 7.62 4.35 10.371A14.946 14.946 0 0024.751 88H67v-9.778H24.751a4.982 4.982 0 01-3.5-1.432 4.858 4.858 0 01-1.45-3.457V53.29a9.696 9.696 0 00-1.878-5.732A9.894 9.894 0 0013 44a9.893 9.893 0 004.924-3.557 9.696 9.696 0 001.878-5.732V.84a14.876 14.876 0 00-5.55 3.457 14.575 14.575 0 00-4.35 10.37v18.09c0 .963-.193 1.916-.566 2.806a7.327 7.327 0 01-1.61 2.379 7.431 7.431 0 01-2.408 1.59c-.901.368-1.867.558-2.842.558H1a1 1 0 00-1 1v5.822a1 1 0 001 1h1.475c1.97 0 3.858.773 5.25 2.148a7.288 7.288 0 012.176 5.186zM152.067 55.244v18.09c0 3.89-1.569 7.62-4.364 10.37A15.018 15.018 0 01137.168 88H97.202v-9.778h39.966a5.007 5.007 0 003.512-1.432 4.852 4.852 0 001.455-3.457V53.29a9.667 9.667 0 011.884-5.732A9.925 9.925 0 01148.958 44a9.925 9.925 0 01-4.939-3.557 9.667 9.667 0 01-1.884-5.732V14.667c0-1.297-.524-2.54-1.455-3.457a5.007 5.007 0 00-3.512-1.432H85.5C83.5 5.867 79 1.63 77 0h60.168c3.952 0 7.741 1.545 10.535 4.296a14.548 14.548 0 014.364 10.37v18.09c0 .963.193 1.916.567 2.806.375.89.923 1.698 1.615 2.379a7.46 7.46 0 002.417 1.59 7.555 7.555 0 002.851.558H161a1 1 0 011 1v5.822a1 1 0 01-1 1h-1.483a7.51 7.51 0 00-5.268 2.148 7.276 7.276 0 00-2.182 5.185z\" fill=\"#08090A\"></path><path d=\"M49.2 0c14.842 0 27.437 9.58 31.886 22.866-7.2 6.092-11.952 14.945-12.61 24.912H63.6c-18.557 0-33.6-14.974-33.6-33.445V1a1 1 0 011-1h18.2zM129 21.111a1 1 0 011 1v10.945c0 17.152-13.968 31.055-31.2 31.055h-12V88h-9.6V54.556l.091-4.778C78.515 33.744 91.98 21.11 108.4 21.11H129z\" fill=\"#1AB3A6\"></path></svg>"
```

Comment:
It's a bit odd that this is a `.txt` file when it contains only an `<svg>` (especially given [how it is used](https://github.com/forem/forem/pull/10745/files#diff-88c874426c0a935a4a1eead5be1314ebR7)). Can we put it inside of `spec/fixtures/icons` or `/spec/fixtures/images` and save it as an `.svg`? We probably should format this correctly and make sure it is optimized, too.

Alternatively, you could just inline the `<svg>` tag in the spec file itself.